pisound-btn/pisound-btn.c

```diff
@@ -79,27 +75,29 @@ static const char *const PISOUND_VERSION_FILE      = "/sys/kernel/pisound/versio
 
 static const char *const UPDATE_CHECK_DISABLE_FILE = BASE_PISOUND_DIR "/disable_update_check"; // If the file exists, the update check will be disabled.
 
-static const char *const DEFAULT_DOWN              = BASE_SCRIPTS_DIR "/system/down.sh";
```

Comment:
The default scripts should not be removed, especially for down and up, as it will break LED blinking when the button is down.